ddd-koans/Value Object/Extra Task - Sealed Class & Functional Approach/task.html

```diff
@@ -0,0 +1,25 @@
+<html>
+<p>
+    You may have noticed that in Self Validation task we throw exception -> this behaviour breaks the flow. <br/>
```

Comment:
`this behaviour breaks the flow.` worth showing how the flow is broken, we can write even a simple main function to indicate that